docs/articles/documentation/recipes/running-tests-with-saucelabs-on-travis-ci/index.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: docs
+title: Running Tests Using Travis CI and Sauce Labs
+permalink: /documentation/recipes/running-tests-using-travis-ci-and-sauce-labs/
+---
+# Running Tests Using Travis CI and Sauce Labs
+
+You can automatically run tests as a part of your build process using [Travis CI](https://travis-ci.org/).
+Furthermore, Travis CI integrates with [Sauce Labs](https://saucelabs.com/) that provides access to multiple real browsers in the cloud (see [Using Sauce Labs with Travis CI](https://docs.travis-ci.com/user/sauce-connect/)).
+Thus you can run your tests completely in the cloud.
+
+Suppose you have a GitHub project, you [installed TestCafe](../using-testcafe/installing-testcafe.md) to that project and created tests. Now you need to automatically run these tests in the cloud when your project is modified.
+To do this, go through the following steps:
+
+* [Step 1 - Enable Travis for your project](#step-1---enable-travis-for-your-project)
+* [Step 2 - Configure Travis to use Sauce Labs](#step-2---configure-travis-to-use-sauce-labs)
+* [Step 3 - Install the Sauce Labs browser provider plugin](#step-3---install-the-sauce-labs-browser-provider-plugin)
+* [Step 4 - Add the `test` script to package.json](#step-4---add-the-test-script-to-packagejson)
+* [Step 5 - Trigger a Travis CI build](#step-5---trigger-a-travis-ci-build)
+
+The [Example](#example) section describes how to run tests with Travis CI and Sauce Labs for a sample project.
+
+## Step 1 - Enable Travis for your project
+
+1. [Sign in to Travis CI](https://travis-ci.org/auth) with your GitHub account. Travis CI will synchronize your repositories from GitHub. You can see them on your [profile page](https://travis-ci.org/profile).
+2. Еnable Travis CI for a repository you want to build by flicking the switch on.
+
+     ![Enable Travis for a repository](../../../images/travis-step-1-2.png)
+
+3. Add a `.travis.yml` configuration file to the root of your project. This file should contain parameters and commands that instruct Travis CI on how to execute your builds. For more information, see [Customizing the Build](https://docs.travis-ci.com/user/customizing-the-build).
```

Comment:
AFAIK, you can use `instruct` without a preposition:

`instruct Travis CI how to...`

sounds more natural